### PR TITLE
Add the rest of the mod opcodes

### PR DESCRIPTION
**Pull request by masonforest**

### CODE REVIEW

File: lib/evm/stack.ex [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# credo:disable-for-this-file Credo.Check.Refactor.CyclomaticComplexity
```
Comment (masonforest) [inline lib/evm/stack.ex]: I'm open to ideas on how to split out all the opcodes. I think this is okay for now though.

File: lib/evm/utils.ex [hunk 2]
```diff
@@ -3,6 +3,9 @@ defmodule EVM.Utils do
 
   defmacro __using__(_opts) do
     quote do
+      # credo:disable-for-next-line Credo.Check.Readability.MaxLineLength
+      @max_int 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
```
Comment (masonforest) [inline lib/evm/utils.ex]: blarg I tried using `:math.pow(2, 256)` but that outputs `float`s which don't add to `int`s properly ->

https://stackoverflow.com/questions/35778222/erlang-math-pow-why-not-integer-by-default-or-typed-for-floats-ints

**Comment by ayrat555** [comment]
I just began reading the yellow paper so I can't say anything about opcodes. I'll come back to this code in the future I guess. 

The code looks ok to me. But maybe we should move every module under Exthereum namespace so it will look like this:

```
module Exthereum.MyModuleName do
...
end
```
```
module Exthereum.EVM.ModuleName do
...
end
```
@masonforest what do you think?

**Comment by masonforest** [comment]
Yep! Good call. I made [another PR](https://github.com/masonforest/exthereum/pull/3).

I skimmed the yellow paper but am mostly using the other implementations as reference:

Here's [pythetherum's opcode implementations](https://github.com/ethereum/pyethereum/blob/develop/ethereum/vm.py#L266) and [ethereumjs's](https://github.com/ethereumjs/ethereumjs-vm/blob/master/lib/opFns.js)

**Pull request merged**